nodeup/pkg/model/tests/containerdbuilder/flatcar/tasks.yaml

```diff
@@ -1,22 +1,10 @@
-contents: |
-  {
-      "cniVersion": "0.4.0",
-      "name": "k8s-pod-network",
-      "plugins": [
-          {
-              "type": "ptp",
-              "ipam": {
-                  "type": "host-local",
-                  "ranges": [[{"subnet": "{{.PodCIDR}}"}]],
-                  "routes": [{"dst":"0.0.0.0/0"}]
-              }
-          },
-          {
-              "type": "portmap",
-              "capabilities": {"portMappings": true}
-          }
-      ]
-  }
+contents: "{\n    \"cniVersion\": \"1.0.0\",\n    \"name\": \"k8s-pod-network\",\n
```

Comment:
Ouch ... can we keep the formatting?  (Did it change because the value is shorter or something like that?  I love yaml!)

Reply:
I'll look in to this in a different PR.